source/issues-and-help.txt

```diff
@@ -0,0 +1,50 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant MongoDB C#/.NET community
+that includes users with varying levels of experience using
+the driver. We find the quickest way to get support for general
+questions is through the `MongoDB Community Forums
+<https://www.mongodb.com/community/forums>`__.
+
+To learn more, refer to our :manual:`support channels </support>`.
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or to request a new feature in the C#/.NET driver,
+please open a case in our issue management tool, JIRA:
+
+* `Create an account and login <https://jira.mongodb.org>`_.
```

Comment:
I think what you are saying is "log in" after creating an account rather than creating a login. Or if logging in is automatic, I don't think you need to list it.

A couple alternatives:

```suggestion
* `Create an account and log in <https://jira.mongodb.org>`_.
```

```suggestion
* `Create a JIRA account <https://jira.mongodb.org>`_.
```